testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveStorageFormats.java

```diff
@@ -534,6 +534,7 @@ public void testTimestampCreatedFromHive(StorageFormat storageFormat)
     }
 
     @Test(dataProvider = "storageFormatsWithNanosecondPrecision")
+    @Flaky(issue = "todo", match = "Failed to translate Hive view")
```

Comment:
The match shouldn't be on `Failed to translate Hive view`. That can mask bugs in the view translation layer.

Instead match on maybe `Object 'table_dropped' not found within 'hive.test_list_failing_views'`?